content/en/tracing/profiling/_index.md

```diff
@@ -74,10 +74,10 @@ The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling l
 
 The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python 3.5+. To begin profiling applications:
 
-1. Install `ddtrace` with the `profile` flavor, which contains both tracing and profiling:
+1. Install `ddtrace`:
 
     ```shell
-    pip install ddtrace[profiling]
+    pip install ddtrace
     ```
 
      **Note**: Profiling is available in the `ddtrace` library for versions 0.36+.
```

Comment:
Should we update this as well? Assuming that `install dd-trace` will only work from a newer version than 0.36,

Reply:
`pip install ddtrace` will install 0.38 (or whatever is newer).

Reply:
I think Gaurab and I are basically asking the same thing :-)

Reply:
Then I'm going to end up writing:
```
If you want to install an older version (< 0.38) of ddtrace, you need to use:

  pip install ddtrace[profiling]
```

Is that what we want? We're in general pretty eager for people to upgrade.